armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -122,7 +127,40 @@ def tearDown(self):
         self.directoryChanger.close()
 
 
+def getInterfaceMocked(self, name=None, function=None):
```

Comment:
This could use a one-line docstring. #shrug

Reply:
https://github.com/terrapower/armi/pull/1308/commits/2eeaa34bdd0c57dbb82cccbbf3d186859cecdcfb

Ended up being unused so I popped out the whole method